_posts/2021-04-01-how-and-when-to-use-ruby-case-statements.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title:  "How and when to use Ruby case statements"
+date:   2021-04-01 12:00:00 +0100
+author: "Shadi Rezek"
+categories: techtips ruby
+og_image: ""
```

Comment:
If you don't want to use an image please remove `og_image`.
Right now it will render `<img src="" .../>` with empty src value on the list of all articles https://docs.knapsackpro.com/

When `og_image` is not present then `<img src="" ../>` won't be rendered.